### PR TITLE
Dev: Cygwin: add xterm instructions

### PR DESCRIPTION
**Pull request by IamPete1**
This adds xterm instructions of the Cygwin wiki page. I was trying to be fancy and do this with the online editor however it turned out bigger than expected so I will do a local test build soon.

### CODE REVIEW

**Comment by IamPete1** [comment]
https://github.com/ArduPilot/ardupilot/pull/14376#event-3345046575

**Comment by IamPete1** [comment]
Just tested the local build, it looks OK to me, must have got lucky!  @stephendade

File: dev/source/docs/building-setup-windows-cygwin.rst [hunk 1]
```diff
@@ -153,3 +154,12 @@ You should now be able to start the "Cygwin64 Terminal" application from your Wi
       The build may fail if the file path to some files is too long.  If the build fails, please try :ref:`cloning <git-clone>` ArduPilot into a directory very high in the directory structure (i.e. ~/ardupilot).
       
       If during build process you get error of missing packet xyz, re-check that you have installed all cygwin packages listed above
+      
+Running SITL
+============
+
+You can run SITL just as you can with any other build envoroment. In order to see the terminal you should instal VcXsrv or other X server aplication. You must then tell Cygwin where to look for the display by seting the DISPLAY varable.
```
Comment (stephendade) [inline dev/source/docs/building-setup-windows-cygwin.rst:161]: A few spelling typos: instal = install, seting = setting, varable = variable

Comment (IamPete1) [reply dev/source/docs/building-setup-windows-cygwin.rst]: I guess google spell check doesn't work when editing on github... fixed

File: dev/source/docs/building-setup-windows-cygwin.rst [hunk 2]
```diff
@@ -153,3 +154,12 @@ You should now be able to start the "Cygwin64 Terminal" application from your Wi
       The build may fail if the file path to some files is too long.  If the build fails, please try :ref:`cloning <git-clone>` ArduPilot into a directory very high in the directory structure (i.e. ~/ardupilot).
       
       If during build process you get error of missing packet xyz, re-check that you have installed all cygwin packages listed above
+      
+Running SITL
+============
+
+You can run SITL just as you can with any other build environment. In order to see the terminal you should instal VcXsrv or other X server application. You must then tell Cygwin where to look for the display by setting the DISPLAY variable.
```
Comment (tridge) [inline dev/source/docs/building-setup-windows-cygwin.rst:161]: spelling "install"

**Comment by Hwurzburg** [comment]
merging after spelling fix

**Comment by IamPete1** [comment]
@Hwurzburg Thanks for fixing this for me

**Pull request merged**